### PR TITLE
fix (ui): fix color of waiting state in ui

### PR DESCRIPTION
**Pull request by bnjjj**
I think orange isn't good choice for "waiting" state. When I've tried the new ui with a pipeline and saw this color I thought that my pipeline was broken. Maybe blue is more appropriate ?

### CODE REVIEW

File: ui/src/app/shared/status/status.icon.html [hunk 1]
```diff
@@ -6,5 +6,5 @@
     <i class="remove red icon" *ngSwitchCase="'Fail'"></i>
     <i class="ban grey icon" *ngSwitchCase="'Disabled'"></i>
     <i class="ban grey icon" *ngSwitchCase="'Skipped'"></i>
-    <i class="wait orange icon" *ngSwitchCase="'Waiting'"></i>
+    <i class="wait blue icon" *ngSwitchCase="'Waiting'"></i>
```
Comment (sguiheux) [inline ui/src/app/shared/status/status.icon.html]: Must override .blue.icon ( semantic icon color) to use $cds_color_teal  => style.scss
Missing TU impact: status.spec.ts and workflow.spec.ts

Comment (yesnault) [inline ui/src/app/shared/status/status.icon.html]: Building is blue. Waiting have to be another color. We can update for another color, but update have to be done on UI2 & CLI (cds ui, then "m")

Comment (sguiheux) [inline ui/src/app/shared/status/status.icon.html]: I think your pb is with CLI. But in WebUI i d'ont think it's a bad idea. 
Waiting: blue clock icon
Building: blue loader

Comment (yesnault) [inline ui/src/app/shared/status/status.icon.html]: ok. We will update cli after if necessary

**Pull request merged**